app/opinions/getNewestOpinions.js

```diff
@@ -0,0 +1,42 @@
+import express from "express";
+import bodyParser from "body-parser";
```

Comment:
You don't need `body-parser`, you can just use `express.json()` and `express.urlencoded()` middlewares, but I don't think that you need urlencoded middleware if you don't use `content-type: application/x-www-form-urlencoded`.